Calibration/EcalAlCaRecoProducers/src/SelectedElectronFEDListProducer.cc

```diff
@@ -566,6 +569,35 @@ void SelectedElectronFEDListProducer<TEle,TCand>::pixelFedDump( std::vector<Pixe
  return ;
 }
 
+template< typename TEle, typename TCand>
+void SelectedElectronFEDListProducer<TEle,TCand>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<vector<edm::InputTag>>("electronTags",{edm::InputTag("hltEgammaGsfElectrons")});
+  desc.add<vector<edm::InputTag>>("recoEcalCandidateTags",{edm::InputTag("hltL1EG25Ele27WP85GsfTrackIsoFilter")});
+  desc.add<edm::FileInPath>("ESLookupTable",edm::FileInPath("EventFilter/ESDigiToRaw/data/ES_lookup_table.dat"));
```

Comment:
@diguida, I have just sent an email to ECAL DB experts. I will let you know asap and, in case, I will update the PR.